src/rosdep2/platforms/redhat.py

```diff
@@ -77,13 +87,13 @@ def register_rhel(context):
     context.set_os_version_type(OS_RHEL, lambda self: self.get_version().split('.', 1)[0])
 
 
-def register_centos(context):
-    # CentOS is an alias for RHEL. If CentOS is detected and it's not set as
-    # an override force RHEL.
+def register_rhel_clone(context, OS_RHEL_CLONE):
```

Comment:
```suggestion
def register_rhel_clone(context, os_rhel_clone_name):
```

I started to flag this as failing to import the constant. But then I realized the all caps was just an argument. It's a lot more readable to make this lowercase so the reader knows it's not a constant.

Reply:
This is a completely reasonable point.

Applied in dd7a48b.